README.md

```diff
@@ -85,6 +85,7 @@ The distributed sessions module can be configured through the environment variab
 | session.DatastoreStore.namespace    |  Namespace to use in the Datastore.                         |  tomcat-gcp-persistent-session |
 | session.DatastoreStore.sessionKind  |  Name of the entity used to store sessions in the Datastore. |  TomcatGCloudSession |
 | session.DatastoreStore.sessionMaxInactiveTime |  Defines the maximum time (in seconds) a session can be inactive before being deleted by the expiration process. | 3600 |
+| session.DatastoreValve.ignoredUriPattern | [Pattern](https://docs.oracle.com/javase/8/docs/api/java/util/regex/Pattern.html) specifying which Uri to ignore when persisting sessions. | null |
```

Comment:
Specify that it's a regex pattern and give an example somewhere.
Also, in the description use "URI" (all caps).

I think `session.datastore.uriExcludePattern` is a better name. Also, I think the other variables should be renamed to `session.datastore.namespace`, etc. "DatastoreStore" sounds odd.